php/example_code/ec2/EC2Service.php

```diff
@@ -0,0 +1,137 @@
+<?php
```

Comment:
Are there tests for this file?

Reply:
No, since they would only be unit tests and those are optional for PHP.